cli/tools/lint/reporters.rs

```diff
@@ -175,6 +175,7 @@ struct JsonLintReporter {
   version: u8,
   diagnostics: Vec<JsonLintDiagnostic>,
   errors: Vec<LintError>,
+  checked_files: Vec<String>,
```

Comment:
I'm thinking this should be sorted just like diagnostics, you can do it in `close()` method